server/src/html/index.html

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
Missing !DOCTYPE. Don't know if necessary

Reply:
It is necessary, but at the moment the file is just placeholder for the final UI page. Maybe it is better to fix this later when the page itself will be made.